### PR TITLE
Add tests for remote disablement of APM

### PR DESCRIPTION
**Pull request by emmettbutler**
## Motivation & Changes

In support of the Python implementation of runtime tracing un-instrumentation in https://github.com/DataDog/dd-trace-py/pull/8080, I've implemented a simple black box test of the functionality described in the relevant product briefs.

## Workflow

1. ⚠️ Create your PR as draft ⚠️
2. Work on you PR until the CI passes (if something not related to your task is failing, you can ignore it)
3. Mark it as ready for review
    * Test logic is modified? -> Get a review from RFC owner. We're working on refining the `codeowners` file quickly.
    * Framework is modified, or non obvious usage of it -> get a review from [R&P team](https://dd.enterprise.slack.com/archives/C025TJ4RZ8X)

:rocket: Once your PR is reviewed, you can merge it!

🛟 [#apm-shared-testing](https://dd.enterprise.slack.com/archives/C025TJ4RZ8X) 🛟

## Reviewer checklist

* [ ] [Relevant label](https://github.com/DataDog/system-tests/blob/main/docs/CI/labels.md) (`run-parametric-scenario`, `run-profiling-scenario`...) are presents
* [ ] No system-tests internal is modified. Otherwise, I have the approval from [R&P team](https://dd.enterprise.slack.com/archives/C025TJ4RZ8X)
* [ ] CI is green, or failing jobs are not related to this change (and you are 100% sure about this statement)
* [ ] A docker base image is modified?
    * [ ] the relevant `build-XXX-image` label is present
    * [ ] To R&P team: locally build and push the image to hub.docker.com 
* [ ] A scenario is added (or removed)?
    * [ ] Get a review from [R&P team](https://dd.enterprise.slack.com/archives/C025TJ4RZ8X)
    * [ ] Once merged, add (or remove) it in system-test-dasboard nightly

### CODE REVIEW

File: utils/parametric/spec/remoteconfig.py
Comment (Kyle-Verhoog) [inline utils/parametric/spec/remoteconfig.py:32]:
```suggestion
    APM_TRACING_ENABLED = 19
```

**Review by cbeauchesne — CHANGES_REQUESTED** [review]
New test is failing.

**Review by cbeauchesne — APPROVED** [review]
AGTM

Though, I did not reviwed the test logic, it may be a good idea to get a review from someone familiar with the tested feature.

File: tests/parametric/test_dynamic_configuration.py
Comment (Kyle-Verhoog) [inline tests/parametric/test_dynamic_configuration.py:416]: I think we should break the test out into its own class so that libraries can pin the precise version that this was introduced rather than grouping it with the other settings

**Comment by cbeauchesne** [comment]
@emmettbutler  for next PR, could you check CI before merging ?  this PR has been merged while the CI was failing on the new test, and in consequence, CI of dd-trace-java, dd-trace-rb and dd-trace-php was broken (tks @robertomonteromiguel for #2060).

For more context, we did not added strict rules on this repo, to keep our agility (flakiness, and also failures related to something else happens quite often), so as long as a single approval at any moment exists, we can merge. But it requires an extra attention when the CI is red.

**Pull request merged**
